platform/marvell-armhf/sonic-platform-et6448m/inband_mgmt.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+#inband_mgmt
+
+inband_mgmt(){
+
+# The intent of this sequence is to ensure 12c bus enumeration order by
+# controlling the order in which the various i2c device drivers are
+# loaded. Hence the drivers are unloaded and then reloaded in the prescribed
+# order.
+# NOTE: In the nokia platform the following sequence is performed by the Nokia
+# platform service init script and thus should not be performed here
+ grep ^onie_platform /host/machine.conf 2>/dev/null | grep nokia >/dev/null
+ if [ $? != 0 ]; then
+     rmmod i2c-dev
+     rmmod i2c_mux_gpio
+     rmmod i2c_mv64xxx
+     modprobe i2c_mv64xxx
+     modprobe i2c-dev
+     modprobe i2c_mux_gpio
+     sleep 60
+ fi
+ while :; do
+   ip -br link show eth0 2> /dev/null
+   if [ $? -eq 0 ]; then
+       ip address show eth0 | grep -qw "inet" 2>/dev/null
+       if [ $? -ne 0 ]; then
+           ifconfig eth0 down
+           systemctl restart networking
+       fi
+       sleep 120
```

Comment:
such sleep concerns me. why do we need so long wait time.

Reply:
At any time user can restart syncd docker by any means, at that time this loop has to monitor the reincarnation of the data port used for in-band and then restart ifup scripts for management port.
Longer the delay lesser the cpu utilization.

Reply:
what ip addr here? are we assume dhcp on eth0? why do we need to wait for ip address to be assign on eth0?

Reply:
In the same board, via Ethernet USB dongle, management interface can also be created. so one difference would be ip address would not be assigned for in-band interface as it would have been created after syncd  init.
Whereas for external eth0, ip address would be assigned statically/dhcp, and it should not be disturbed unnecessarily.